src/List/styled.js

```diff
@@ -19,6 +19,17 @@ const orderedProperties = css`
   }
 `;
 
+const horizontalProperties = css`
+  & li {
+    display: inline-block;
+    margin-right: 1rem;
```

Comment:
Lets maybe unapply the default margin-bottom styling we get with vertical lists

```suggestion
    margin-right: 1rem;
    margin-bottom: unset;
```

Reply:
Done 👍